doc/classes/@GlobalScope.xml

```diff
@@ -720,6 +729,23 @@
 				[b]Note:[/b] Errors printed this way will not pause project execution. To print an error message and pause project execution in debug builds, use [code]assert(false, "test error")[/code] instead.
 			</description>
 		</method>
+		<method name="push_error_formatted">
+			<return type="void">
+			</return>
+			<argument index="0" name="message" type="String">
+			</argument>
+			<argument index="1" name="error" type="int">
+			</argument>
+			<description>
+				Pushes an error message along with a human readable name of the error code to Godot's built-in debugger and to the OS terminal.
+				[codeblock]
+				var file = File.new()
+				var err = file.open("res://file.txt", File.READ)
+				if err:
+				    push_error_formatted("failed to open file", err) # Prints, e.g. "failed to open file: File not found", to debugger and terminal as error call
```

Comment:
The line is pretty long. It's better to split the comment to put it on the line above. Also, error messages should preferably start with a capital letter:

```suggestion
					# Prints, e.g. "Failed to open file: File not found", to debugger and terminal as error call
				    push_error_formatted("Failed to open file", err)
```